packages/@react-aria/slider/src/useSlider.ts

```diff
@@ -173,8 +173,18 @@ export function useSlider(
       ...fieldProps
     },
     trackProps: mergeProps({
-      onMouseDown(e: React.MouseEvent<HTMLElement>) { onDownTrack(e, undefined, e.clientX, e.clientY); },
-      onPointerDown(e: React.PointerEvent<HTMLElement>) { onDownTrack(e, e.pointerId, e.clientX, e.clientY); },
+      onMouseDown(e: React.MouseEvent<HTMLElement>) {
+        if (e.button !== 0) {
+          return;
+        }
+        onDownTrack(e, undefined, e.clientX, e.clientY);
+      },
+      onPointerDown(e: React.PointerEvent<HTMLElement>) {
```

Comment:
Do we even need the pointer event if we have mouse events and touch events already?

Reply:
well, we should probably get rid of it with the rest of the pointer events?

Reply:
looks like we need it otherwise useMove's pointer event eats the mouseDown event and onMouseDown never triggers